src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4872,4 +4872,16 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureTuples" xml:space="preserve">
     <value>tuples</value>
   </data>
+  <data name="ERR_AmbiguousDeconstruct" xml:space="preserve">
+    <value>More than one Deconstruct instance or extension method was found for type '{0}'.</value>
+  </data>
+  <data name="ERR_DeconstructRequiresOutParams" xml:space="preserve">
+    <value>The Deconstruct method for type '{0}' must have only out parameters.</value>
```

Comment:
Rather than referring to the type, would it be more informative to refer to the method, so that it would be clear what method was picked by the compiler?